src/components/RespondentsTable/RespondentsTable.tsx

```diff
@@ -69,8 +74,28 @@ export const RespondentsTable = (): JSX.Element => {
         value: lastEdited,
       },
       actions: {
-        content: () => '',
+        content: () => (
+          <StyledFlexTopCenter>
+            <StyledSvg
+              id="dots"
+              width={18}
+              height={4}
+              onMouseEnter={() => setShowActions(true)}
+              onMouseLeave={() => setShowActions(false)}
+            />
+            {showActions && (
+              <StyledActions>
```

Comment:
Could you pls create separate component for actions or at lest move styles to styles/styledComponents/Actions.ts to avoid duplicate the code in different table components. Pls also change inheritance of actions in the AppletTable.

Reply:
Also, it would be good to move three dots to the new Actions component.